### PR TITLE
add security policy

### PR DESCRIPTION
**Pull request by maci3jka**
This pr adds security.md and introduces a security policy enabling users to report security issues

KU-2046

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported versions
+
+Support for fallows lifecycle mentioned in [Canonical Kubernetes release cycle](https://ubuntu.com/about/release-cycle#canonical-kubernetes-release-cycle).
```
Comment (ktsakalozos) [inline security.md:5]: This "Support for fallows" does not read right.

Comment (maci3jka) [reply security.md:5]:
```suggestion
Support follows the lifecycle mentioned in [Canonical Kubernetes release cycle](https://ubuntu.com/about/release-cycle#canonical-kubernetes-release-cycle).
```

**Review by ktsakalozos — CHANGES_REQUESTED** [review]
Left you a minor comment.

**Comment by maci3jka** [comment]
> Left you a minor comment.

updated

**Review by ktsakalozos — APPROVED** [review]
LGTM +1

**Comment by ktsakalozos** [comment]
The errors are not related to this PR

**Pull request merged**